### PR TITLE
Fixes karaf:run goal for single bundle execution

### PR DESCRIPTION
**Pull request by jozzan**
See issue: https://issues.apache.org/jira/browse/KARAF-5630

### CODE REVIEW

**Comment by gnodet** [comment]
retest this please

File: tooling/karaf-maven-plugin/src/main/java/org/apache/karaf/tooling/RunMojo.java [hunk 1]
```diff
@@ -141,6 +145,7 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
             Object featureService = findFeatureService(bundleContext);
             addFeatureRepositories(featureService);
+            fixProjectArtifact();
```
Comment (jbonofre) [inline tooling/karaf-maven-plugin/src/main/java/org/apache/karaf/tooling/RunMojo.java]: I would avoid to introduce such method name here. Why not simply adding a comment with the code inline ?

File: tooling/karaf-maven-plugin/src/main/java/org/apache/karaf/tooling/RunMojo.java [hunk 2]
```diff
@@ -153,6 +158,14 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         }
     }
 
+    void fixProjectArtifact() {
+        Artifact artifact = project.getArtifact();
+
+        if(artifact.getFile() == null || !artifact.getFile().exists()) {
+            artifact.setFile(new File(bundleArchivePath));
```
Comment (jbonofre) [inline tooling/karaf-maven-plugin/src/main/java/org/apache/karaf/tooling/RunMojo.java]: Sorry, but I didn't understand the purpose here. `karaf:run` is supposed to be in an execution cycle, so it should not attach any artifact.

**Comment by jbonofre** [comment]
See my comment in Jira, it already works as expected.

**Pull request closed without merging**